airbyte-integrations/connectors/source-github/setup.py

```diff
@@ -16,7 +16,7 @@
     author_email="contact@airbyte.io",
     packages=find_packages(),
     install_requires=MAIN_REQUIREMENTS,
-    package_data={"": ["*.json", "schemas/*.json", "schemas/shared/*.json"]},
+    package_data={"": ["**/*.json"]},
```

Comment:
This was an accidental commit from another project. Reverting...

```suggestion
    package_data={"": ["*.json", "schemas/*.json", "schemas/shared/*.json"]},
```